src/sentry/tasks/store.py

```diff
@@ -417,7 +417,11 @@ def save_attachment(event, attachment):
     file.putfile(six.BytesIO(attachment.data))
 
     EventAttachment.objects.create(
-        event_id=event.event_id, project_id=event.project_id, name=attachment.name, file=file
+        event_id=event.event_id,
+        group_id=event.group_id,
```

Comment:
Does this work with transactions? Which means: can we upload attachments to transactions ? For those event.group will be None

Reply:
It should since we made group id column nullable